### ISSUE TITLE
While training zip file, "Seems that the dataset is not valid""

### ISSUE THREAD

**Issue by sandip-narwade** [alessiosavi/PyRecognizer#18]
I am adding a zip file for training but throwing below error

```json
{
  "data": null,
  "date": "2020-01-09 13:42:44.162639",
  "description": "Seems that the dataset is not valid",
  "error": "ERROR DURING LOADING DAT",
  "status": "KO"
}
```

Logfile:  
```text
2020-01-09 13:42:43,766 - root - WARNING | [Administrator.py:24] | Initializing a new redis connection
2020-01-09 13:42:44,105 - root - DEBUG | [Api.py:110] | train_network | Starting training phase ...
2020-01-09 13:42:44,105 - root - DEBUG | [util.py:246] | retrieve_dataset | Parsing dataset ...
2020-01-09 13:42:44,105 - root - DEBUG | [util.py:219] | verify_extension | File: **face_training_dataset_little.zip** | Ext: .zip
2020-01-09 13:42:44,106 - root - INFO | [util.py:222] | Verifying zip bomb ...
2020-01-09 13:42:44,162 - root - DEBUG | [util.py:261] | tune_network | Dataset parsed!
2020-01-09 13:42:44,162 - root - DEBUG | [Response.py:30] | parse_data | Parsing None
2020-01-09 13:42:44,166 - werkzeug - INFO | [_internal.py:122] | 192.168.15.46 - - [09/Jan/2020 13:42:44] "[37mPOST /train HTTP/1.1[0m" 200 -
```

**Comment by alessiosavi** [comment]
Please, run the `tree` GNU command in the unzipped folder and post the result. If the data are not sensitive, upload the data and share the link.

**Comment by sandip-narwade** [comment]
I used [face_training_dataset_little.zip](https://github.com/alessiosavi/PyRecognizer/blob/master/dataset/face_training_dataset_little.zip)

**Comment by alessiosavi** [comment]
Hi @sandip-narwade, thank you very much for the issue: you have found a bug!

Fix in e45916d2a3eb8d8ffe2ae0cb6a709d217a64d2ed v0.3.2
